build_run_specs_full.py

```diff
@@ -0,0 +1,237 @@
+entries = [
+    #bigbench
+    # 1. auto_debugging: https://github.com/google/big-bench/tree/main/bigbench/benchmark_tasks/auto_debugging
```

Comment:
@msaroufim  @artidoro, how you feel about code understanding type of tasks for auto_debugging and code description task?